index.js

```diff
@@ -2,17 +2,44 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
+// added defer to the script tag 
 
 /***** Deliverable 2 *****/
+const newHeader = document.querySelector('#header')
 
+newHeader.style.color = 'red'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const firstNewImg = document.querySelector("#profile img")
+firstNewImg.src = traveler.photo
+
+firstNewImg.alt = traveler.name 
+
+console.log(firstNewImg)
+
+const travelerName = document.querySelector("#profile > h2")
+travelerName.innerText = traveler.name 
+
+const travelerNickName = document.querySelector("#profile > em")
+travelerNickName.innerText = traveler.nickname 
+
+
+
+
+    
 
 
 /***** Deliverable 4 *****/
 
+function renderSightings(animalSightings){
+    const outerLi = document.createElement('li')
```

Comment:
This is the right track~! Think about when we need to create these `li` tags. We need to do it **for eachhhhhhhh** element in our `traveler` array. How could we maybe do that??  

Once you're there, you have several options for executing this. Think about what we'd need to accomplish this! (hint, innerHTML can help you here.)